samples/xti/codereadercallback.h

```diff
@@ -1,4 +1,4 @@
-/////////////////////////////////////////////////////////////////////////////
+﻿/////////////////////////////////////////////////////////////////////////////
```

Comment:
It looks like this change adds a BOM to the file, which is unwanted. Could you please get rid of it? TIA!